_component_design/badge.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Badges
+parent: Notifications
+layout: component
+category: Components
+usage: >
+  Notification badges are used to let the user know that there is new activity that hasn’t yet been acknowledged. It is used to encourage them to click into that section to see the details of the activity. 
```

Comment:
Use notification badges to alert the user that there is new activity they have not acknowledged yet. Use them to encourage the user to go to that section to see the details of the activity.

Reply:
updated.